contracts/contracts/flipper/Flipper.sol

```diff
@@ -0,0 +1,154 @@
+pragma solidity 0.5.11;
+
+import "../governance/Governable.sol";
+import "../token/OUSD.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+// Contract to exchange usdt, usdc, dai from and to ousd.
+//   - 1 to 1. No slippage
+//   - Optimized for low gas usage
+//   - No guarantee of availability
+
+interface Tether {
+    function transfer(address to, uint256 value) external;
+
+    function transferFrom(
+        address from,
+        address to,
+        uint256 value
+    ) external;
+
+    function balanceOf(address) external returns (uint256);
+}
+
+contract Flipper is Governable {
+    using SafeERC20 for IERC20;
+
+    uint256 constant MAXIMUM_PER_TRADE = (25000 * 1e18);
```

Comment:
What's the reasoning behind this maximum? Trying to think through the advantage of it. If I wanted to circumvent it I'd just run multiple buys or sells in one transaction.